src/components/CharacterItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './CharacterItem.css';
+
+class CharacterItem extends React.Component {
```

Comment:
This can be just a FC, you don't have any state in it.

Reply:
done